lambda_layers/core_layer/python/core_layer/model/url_model.py

```diff
@@ -19,3 +20,4 @@ class ItemURL(Base):
     item = relationship("Item", back_populates="urls")
     url_id = Column(String(36), ForeignKey('urls.id'))
     url = relationship("URL", back_populates="items")
+    unsafe = Column(String(50))
```

Comment:
Why does the mapping table need this attribute? Wouldn't it be enough to only have this in the url model?

Reply:
I wasn't sure about this. In my opinion it is redunant, however I find the whole ItemUrl and Url structure redunant. I couldn't decide on which level it's better. 
For me it is fine to remove it for ItemUrl. I will remove ItemUrl#unsafe and remove it from alembic modified sql as well).

Reply:
Thanks. We need the mapping table, to realize n-to-m relations (One url can be included in many items and one item can have multiple links).

Reply:
Done